runtime/src/lib.rs

```diff
@@ -380,6 +428,163 @@ impl anchor::Trait for Runtime {
+/// Adding, removing, swapping, resetting members requires an approval of simple majority of the Council
+/// or `Root` origin, the technical committee itself cannot change its membership
+impl pallet_membership::Trait<pallet_membership::Instance2> for Runtime {
+    type Event = Event;
+    type AddOrigin = RootOrMoreThanHalfCouncil;
+    type RemoveOrigin = RootOrMoreThanHalfCouncil;
+    type SwapOrigin = RootOrMoreThanHalfCouncil;
+    type ResetOrigin = RootOrMoreThanHalfCouncil;
+    type PrimeOrigin = RootOrMoreThanHalfCouncil;
+    type MembershipInitialized = TechnicalCommittee;
+    type MembershipChanged = TechnicalCommittee;
+}
+
+parameter_types! {
+    pub const MaxScheduledPerBlock: u32 = 50;
+}
+
+impl pallet_scheduler::Trait for Runtime {
+    type Event = Event;
+    type Origin = Origin;
+    type PalletsOrigin = OriginCaller;
+    type Call = Call;
+    type MaximumWeight = MaximumBlockWeight;
+    type ScheduleOrigin = EnsureRoot<AccountId>;
+    type MaxScheduledPerBlock = MaxScheduledPerBlock;
+    type WeightInfo = ();
+}
+
+parameter_types! {
+    pub const EnactmentPeriod: BlockNumber = 2 * DAYS;
+    pub const LaunchPeriod: BlockNumber = 20 * DAYS;
+    pub const VotingPeriod: BlockNumber = 15 * DAYS;
+    pub const CooloffPeriod: BlockNumber = 1 * DAYS;
+    pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
+    // 10K tokens
+    pub const MinimumDeposit: Balance = 10_000 * 1_000_000;
+    // 1 token
+    pub const PreimageByteDeposit: Balance = 1 * 1_000_000;
+    pub const MaxVotes: u32 = 100;
+    pub const MaxProposals: u32 = 100;
+}
+
+impl simple_democracy::Trait for Runtime {
+    type Event = Event;
+    type PublicProposalDeposit = MinimumDeposit;
+    /// Only council members can vote
+    type VoterOrigin = CouncilMember;
+}
+
+impl pallet_democracy::Trait for Runtime {
+    type Proposal = Call;
+    type Event = Event;
+    type Currency = Balances;
+    type EnactmentPeriod = EnactmentPeriod;
+    type LaunchPeriod = LaunchPeriod;
+    type VotingPeriod = VotingPeriod;
+    type CooloffPeriod = CooloffPeriod;
+    type MinimumDeposit = MinimumDeposit;
+    type ExternalOrigin = CouncilMember;
+    type ExternalMajorityOrigin = CouncilMember;
+    type ExternalDefaultOrigin = RootOrMoreThanHalfCouncil;
+    /// Two thirds of the technical committee can have an ExternalMajority/ExternalDefault vote
+    /// be tabled immediately and with a shorter voting/enactment period.
+    type FastTrackOrigin = EnsureOneOf<
+        AccountId,
+        pallet_collective::EnsureProportionAtLeast<_2, _3, AccountId, TechnicalCollective>,
+        EnsureRoot<AccountId>,
+    >;
+    type InstantOrigin = EnsureRoot<AccountId>;
+    type InstantAllowed = ();
+    type FastTrackVotingPeriod = FastTrackVotingPeriod;
+    type CancellationOrigin = RootOrMoreThanHalfCouncil;
+    /// Any council member can cancel a public proposal
+    type CancelProposalOrigin = CouncilMember;
```

Comment:
This doesn't make sense to me. Allowing council members to slash anyone who makes a proposal sounds like a pretty effective avenue for voter suppression.

Reply:
This was to get rid of proposals that obviously wouldn't be acceptable and thus would waste 1 round. The reason I didn't require a majority of Council was due to less availability of the Council. If one member is too less, we can make it to 2 (regardless of Council size) or a smaller fraction.

Reply:
The problem is that people can get slashed for proposing the wrong thing.

Reply:
I am leaving it to the Council member's discretion to decide what is "wrong" enough to be penalized. We can make this action doable by approval of more than 1